common-utils/README.md

```diff
@@ -174,6 +231,71 @@ f"{project_name}-prod-live"
 
 `prod` will be used for flows pushed to the `main-v2` branch.
 
+The purpose of this `deploy-flows` is to:
+
+- Using the `tool.prefect.flows_folder` configuration, find Python files that end in `_flow.py` and attempt to deploy them to Prefect.
+- The deployment of a flow consists of:
+    - Creating an ECS Task Block in Prefect that defines the flow's task definition.
+    - Loading the flow files to the S3 filessystem.
+    - Registering the flow's deployment configurations with Prefect.  A flow can have multiple deployment configurations.  Deployment documentation is found [here](https://docs.prefect.io/concepts/deployments/).
+
+#### **Flow Deployment and Deployment Configurations**
+
+The folder to look for flows is defined in the pyproject.toml.
+
+```toml
+[tool.prefect]
+flows_folder = "tests/test_flows"
+```
+
+Once a flow file is found, we look for a global variable in the file called `FLOW_SPEC`.
+
+This variable must be an instantiated `FlowSpec` object.
+
+```python
+from prefect import flow, task
+from prefect.server.schemas.schedules import CronSchedule
+
+from common.deployment import FlowDeployment, FlowSecret, FlowSpec
+
+
+@task()
+def task_1():
+    print("hello world")
+
+
+@flow()
+def flow_1():
+    task_1()
+
+
+FLOW_SPEC = FlowSpec(
```

Comment:
Would it be worth creating some standard flows specs? Perhaps a `big` and `small` or similar?

Reply:
I think that makes a lot of sense...add to github issues for now as enhancement